### ISSUE TITLE
Offscreen menu should be labelled appropriately and not be accessible when visually hidden 

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#364]
### Description of issue
The offscreen menu is labelled to AT users as "offscreen menu" using an `aria-label` which may not make sense as "offscreen" will not be descriptive/meaningful to folks with low vision or blindness.

The offscreen menu will always be accessible to screen reader users regardless on desktop or mobile as it is only visually hidden as opposed to also being invisible to screen readers by means of `aria-hidden="true"`

### Expected result
Screen reader users should only be able to access one menu/navigation similar to sighted users.

### Actual result
Screen reader users can access both desktop and mobile menus.

### Steps to reproduce
1. [Run this landmarks extension](https://chrome.google.com/webstore/detail/landmark-navigation-via-k/ddpokpbjopmeeiiolheejjpkonlkklgp?hl=en-US)
2. Confirm that several navigations are listed including "offscreen navigation"

```
Path:      /
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  1792 x 952 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1792x952&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/)

**Comment by joshkimux** [comment]
@marcusrelacion forgot to assign you here!

**Comment by marcusrelacion** [comment]
I've removed the library for the offscreen menu to work on a simpler solution.
